site/src/component/ReviewForm/ReviewForm.tsx

```diff
@@ -81,19 +82,48 @@ const ReviewForm: FC<ReviewFormProps> = (props) => {
       if (cookies.user === undefined) {
         alert('You must be logged in to add a review!');
         props.closeForm();
+      } else {
+        setSubmitted(false);
       }
     }
-  }, [showForm, props, cookies]);
+    //If editable is true
+    if (props.review) {
+      const [year, quarter] = props.review.quarter.split(' ');
+      setReviewId(props.review?._id);
+      setQuarterTaken(quarter);
+      setYearTaken(year);
+      setGradeReceived(props.review.gradeReceived);
+      setDifficulty(props.review.difficulty);
+      setQuality(props.review.rating);
+      setContent(props.review?.reviewContent);
+      setSelectedTags(props.review?.tags);
+      setAttendance(props.review?.attendance);
+      setTakeAgain(props.review?.takeAgain);
+      setTextbook(props.review?.textbook);
+      setUserName(props.review?.userDisplay);
+      setProfessor(props.review?.professorID);
+      setCourse(props.review?.courseID);
+    }
+  }, [showForm]);
```

Comment:
Add the dependencies to clear the warning. The lint check will allow warnings but our goal should be to have none so let's make sure not to add anymore.

```suggestion
  }, [showForm, cookies.user, props]);
```